### PR TITLE
fix: repositioning cues by config

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

Issue: can't change cue visibility by config.
Solution: add config and use our API.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: flow-typed/types/playback-config.js [hunk 1]
```diff
@@ -14,5 +16,6 @@ declare type PKPlaybackConfigObject = {
   streamPriority: Array<PKStreamPriorityObject>,
   preferNative: PKPreferNativeConfigObject,
   inBrowserFullscreen: boolean,
-  playAdsWithMSE: boolean
+  playAdsWithMSE: boolean,
+  TextTrackDisplaySetting: Cue
```
Comment (dan-ziv) [inline flow-typed/types/playback-config.js:20]: `textTrackDisplaySetting `

**Review by dan-ziv — CHANGES_REQUESTED** [review]

**Review by dan-ziv — CHANGES_REQUESTED** [review]

File: src/player.js [hunk 2]
```diff
@@ -475,6 +475,7 @@ export default class Player extends FakeEventTarget {
     } else {
       Utils.Object.mergeDeep(this._config, config);
     }
+    this.setTextDisplaySettings(this._config.playback.TextTrackDisplaySetting);
```
[the remark this answers is not in the document]
Comment (Yuvalke) [reply src/player.js]: So we'll be able to clear previous configuration

File: src/player.js [hunk 3]
```diff
@@ -1225,11 +1226,11 @@ export default class Player extends FakeEventTarget {
 
   /**
    * update the text display settings
-   * @param {Object} settings - text cue display settings
+   * @param {Cue} settings - text cue display settings
    * @public
    * @returns {void}
    */
-  setTextDisplaySettings(settings: Object): void {
+  setTextDisplaySettings(settings: Cue): void {
```
[the remark this answers is not in the document]
Comment (OrenMe) [reply src/player.js:1233]: Right, it should either be cue configuration object and in any case it is partial config as we don't request to pass full one.
This whole api is not final so we can continue with object.

**Comment by OrenMe** [comment]
we will do a more general solution on FEC-10766

**Pull request closed without merging**